src/components/panels/SupportProposalPanel.js

```diff
@@ -62,7 +62,7 @@ const SupportProposal = React.memo(function SupportProposal({
   // Amount change handler
   const handleAmountChange = useCallback(
     event => {
-      const newAmount = event.target.value
+      const newAmount = event.target.value.replace(/,/g, '.').replace(/-/g, '')
```

Comment:
Lets only replace the `-` character. 

When users input a `,` the UI already gives feedback about being an invalid amount.